client/components/credit-card-form-fields/index.jsx

```diff
@@ -145,40 +149,48 @@ export class CreditCardFormFields extends React.Component {
 			<div className="credit-card-form-fields">
 				{ this.createField( 'name', Input, {
 					autoFocus,
-					label: translate( 'Cardholder Name', {
+					label: translate( 'Cardholder Name {{nameExplainer/}}', {
```

Comment:
Using `nameExplainer ` component here has two issues:
1. The component makes another translate call, and having nested `translate()` looks like something that can be avoided, just for reduced complexity and cleaner flow.
2. The second translate call is missing a translator comment, so `(as written on card)` could be hard to get the context for a translator. Of course, we could add a comment here, but refer previous point, it makes it more complex.

I would suggest rewriting this as:

```
label: translate( 'Cardholder Name {{span}}(as written on card){{/span}}', {
	comment: 'Cardholder name label on credit card form',
	components: {
		span: <span className="credit-card-form-fields__explainer" />,
	},
} ),
```

The advantage is that _(as written on card)_ is now part of the original string, so a translator gets the full context.

Reply:
Great suggestion! Addressed in https://github.com/Automattic/wp-calypso/pull/33034/commits/25a06704d7abfa647067fcc06631d7846a6c9936.